htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java

```diff
@@ -84,6 +85,20 @@ public void setName(String name) {
     }
 
     /**
+     * Determines whether or not this element exists on page.
+     *
+     * @return True if the element exists on page, false otherwise.
+     */
+    public boolean exists() {
+        try {
+            getWrappedElement().isDisplayed();
+            return true;
```

Comment:
To have `return ` at the method end instead of the middle. It's more a code style issue, rather then something that doesn't work. Also on code coverage report it should look better.